src/sentry/integrations/slack/integration.py

```diff
@@ -34,6 +44,9 @@ def build_integration(self, state):
             'metadata': {
                 'bot_access_token': data['bot']['bot_access_token'],
                 'bot_user_id': data['bot']['bot_user_id'],
+                # XXX: should this be stored with OrganizationIntegration?
```

Comment:
kind of sucks we have to use a user token for this, but i guess it makes sense somewhere in the universe